iree/compiler/Dialect/Modules/TensorList/Conversion/test/convert_hal_to_vm.mlir

```diff
@@ -27,3 +27,21 @@ func @SetItem(%list: !tensorlist.list, %index: !hal.buffer_view, %item: !hal.buf
   return %0 : !tensorlist.list
 }
 // CHECK: vm.import @tensorlist.set_item
+
+// -----
+
+// CHECK-LABEL: @Stack
+func @Stack(%list: !tensorlist.list, %element_shape: !hal.buffer_view, %num_elements: !hal.buffer_view) -> !hal.buffer_view {
```

Comment:
Thanks for agumenting the test coverage for Stack in this CL :)))

Reply:
No problem :P. Was an easy drive by.